install/data-fashion-mnist.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
```

Comment:
should probably also make all the datasets separate docker containers that download and preprocess them... i can give it a shot later (not in this PR)